docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+When a Workflow task is dispatched, Temporal will look at the build ID of the Workflow and send the task to a Worker with a compatible build ID.
+This allows seamless deployment of new code versions without Workflow tasks getting stuck.
+Some key aspects of build ID dispatch:
+
+- Workers advertise their build ID to Temporal when polling for tasks.
+- Task Queues maintain sets of compatible build IDs.
+  New IDs can be added to existing compatible sets.
+- When dispatching a Workflow task, Temporal matches the Workflow's build ID to a compatible Worker build ID.
+- Build IDs allow different versions of code to run side by side on the same task queue.
+- Promoting a new build ID to the default set directs new Workflows to start on that code version.
+
+So in summary, build ID dispatch provides a mechanism to deploy new Worker code without disrupting existing Workflows on a Task Queue.
+It's a key feature for zero-downtime deployments in Temporal.
+
+### Patching APIs
+
+Patching APIs refers to the ability to update a Workflow Definition without breaking existing Workflow Executions.
+
+<!--
+Worker Versioning simplifies the process of deploying changes to [Workflow Definitions](/workflows/#workflow-definition).
+It does this by letting you define sets of versions that are compatible with each other, and then assigning a Build ID to the code that defines a Worker.
+The Temporal Server uses the Build ID to determine which versions of a Workflow Definition a Worker can process.
+-->
+
+## Worker-Specific Task Queues
+
+| Feature                                                     | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Worker-Specific Task Queues](#worker-specific-task-queues) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Worker-Specific Task Queue, also known as Sticky Execution, is when a Worker Entity caches the Workflow in memory and creates a dedicated Task Queue to listen on.
+
+A Worker-Specific Task Queue occurs after a Worker Entity completes the first Workflow Task in the chain of Workflow Tasks for the Workflow Execution.
+
+The Worker Entity caches the Workflow in memory and begins polling the dedicated Task Queue for Workflow Tasks that contain updates, rather than the entire Event History.
+
+If the Worker Entity does not pick up a Workflow Task from the dedicated Task Queue in an appropriate amount of time, the Cluster will resume Scheduling Workflow Tasks on the original Task Queue.
+Another Worker Entity can then resume the Workflow Execution, and can set up its own Worker-Specific Task Queue for future Workflow Tasks.
+
+- [How to set a `StickyScheduleToStartTimeout` on a Worker Entity in Go](/dev-guide/go/foundations#stickyscheduletostarttimeout)
+
+Worker-Specific Task Queues are the default behavior of the Temporal Platform.
+
+## Workflow Execution mechanisms
```

Comment:
This is a confusing title for these features. Temporal supports local activities, but it just so happens a couple older SDKs also have this other concept that doesn't even need to be discussed globally